DTS-STN/Service-Canada-Labs
# Explore Updates Section

This PR covers two tasks

### [Create "Read our latest project updates" section](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215234)
### [Create new "Project updates" section](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=215243)

![Screenshot 2024-06-07 at 3 36 11 PM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/a18b9b29-c65d-4323-a85c-b2885aa17332)

The new "Explore our updates" section has been implemented on the homepage, all project pages, and all article pages.
Updates are always sorted with most recent first.

Created a unit test and story for the new ExploreUpdates.js component.

When the content for the ExploreUpdates and ExploreProjects sections is ready in AEM, we will revisit to remove the hardcoded values for the header on ExploreUpdates on some pages etc.

## Test Instructions

1. `yarn dev` (also do this with `yarn build` + `yarn start`)
2. Navigate to home page, project pages and article pages
3. See that updates section is rendering properly, is sorted by most recent, is mobile responsive, has proper styling, and does not include the current page
